weave/trace_server/methods/evaluation_status.py

```diff
@@ -87,7 +90,7 @@ def get_total_rows() -> int:
         table_rows_stats = server.table_query_stats(
             tsi.TableQueryStatsReq(
```

Comment:
i'm going to put up a fast follow to mark all of our req types with `model_config = ConfigDict(extra="forbid")` that would have caught this....

Reply:
👍 yup good idea